pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/kafka/AvroRecordToPinotRowGenerator.java

```diff
@@ -95,22 +95,4 @@ public GenericRow transform(GenericData.Record record, org.apache.avro.Schema sc
 
     return destination;
   }
-
-  public GenericRow transform(GenericRecord avroRecord, GenericRow destination) {
```

Comment:
Checked. And it's not safe to use because it does not handle null multi-value column case.